packages/trader/src/Modules/Trading/Components/Form/ContractType/contract-type-list.jsx

```diff
@@ -6,7 +6,11 @@ const List = ({ handleInfoClick, handleSelect, list, name, value }) =>
     list.map((contract_category, key) => {
         const contract_types = contract_category.contract_types?.filter(contract_type => {
             const base_contract_type = /^(.*)_equal$/.exec(contract_type.value)?.[1];
+            const is_turbosshort = contract_type.value === 'turbosshort';
 
+            if (is_turbosshort) {
```

Comment:
Doesn't this condition always return `true` if `is_turbosshort` is also `true`?

Reply:
This condition checks if `turbosshort` contract is present in the list and that would be present because we are getting it as a separate tradetype. As this condition passes we will check for the value and omit the `turboshort` from the list.
So the turbos short would be omitted from the Trade Type List.

as this line will give the `false`
`return contract_category.contract_types.some(c => c.value === is_turbosshort)`

Reply:
I mean instead of lines 9-12 couldn't we write `return contract_type.value === 'turbosshort';` ?